service.py

```diff
@@ -119,14 +119,28 @@ def mirror_sub(id, filename, sub_file):
         pass
 
 def takeTitleFromFocusedItem():
-    labelType = xbmc.getInfoLabel("ListItem.DBTYPE")  #movie/tvshow/season/episode
     labelMovieTitle = xbmc.getInfoLabel("ListItem.OriginalTitle")
     labelYear = xbmc.getInfoLabel("ListItem.Year")
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
     isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```

Comment:
Why broken?
See the whole file, every thing is the same, I just move it down few lines and change the condition as CaTz told me.